### ISSUE TITLE
CairoSVG 1.0.21 has TypeError: __init__() got an unexpected keyword argument 'resolve_entities'

### ISSUE THREAD

**Issue by rfloriano** [Kozea/CairoSVG#105]
Running [Thumbor](https://github.com/thumbor/thumbor) tests, I got this error in version 1.0.21.

Traceback:
```
Traceback (most recent call last):
  File "/Users/rfloriano/workspace/python/tornado/thumbor/tests/engines/test_base_engine.py", line 109, in test_convert_svg_with_xml_preamble_to_png
    self.engine.convert_svg_to_png(buffer)
  File "/Users/rfloriano/workspace/python/tornado/thumbor/thumbor/engines/__init__.py", line 147, in convert_svg_to_png
    buffer = cairosvg.svg2png(bytestring=buffer, dpi=self.context.config.SVG_DPI)
  File "/Users/rfloriano/.virtualenvs/thumbor/lib/python2.7/site-packages/cairosvg/__init__.py", line 43, in <lambda>
    surface_type.convert(*args, **kwargs))(_surface_type)
  File "/Users/rfloriano/.virtualenvs/thumbor/lib/python2.7/site-packages/cairosvg/surface/__init__.py", line 103, in convert
    tree = Tree(**kwargs)
  File "/Users/rfloriano/.virtualenvs/thumbor/lib/python2.7/site-packages/cairosvg/parser.py", line 270, in __init__
    parser = ElementTree.XMLParser(resolve_entities=unsafe)
TypeError: __init__() got an unexpected keyword argument 'resolve_entities'
```

**Comment by liZe** [comment]
Oh, yes, this happen when you don't have `lxml` installed :/.

**Comment by liZe** [comment]
Fixed in 1.0.22, thanks!

**Comment by rfloriano** [comment]
You're welcome! @liZe
